backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/EligibilityController.java

```diff
@@ -65,11 +65,18 @@
 
 			logger.info("checkEligibility response: {} ", checkEligibilityResponse);
 			return response;	
+		} catch (HNWebException hwe) {
+			switch (hwe.getType()) {
+			case DOWNSTREAM_FAILURE:
+				throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, hwe.getMessage(), hwe);
+			default:
+				throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Bad /lookup-phn request", hwe);				
```

Comment:
Copy paste errors here should be /check-eligibility

Reply:
Fixed. Plan is to move to a global ExceptionHandler eventually as the exception handling code is getting duplicated and really all our controllers follow a similar pattern.